modulefiles/tasks/gaea/miniconda_regional_workflow.lua

```diff
@@ -1,4 +1 @@
-prepend_path("MODULEPATH", "/lustre/f2/dev/role.epic/contrib/modulefiles")
-load(pathJoin("miniconda3", os.getenv("miniconda3_ver") or "4.12.0"))
-
```

Comment:
miniconda3 should be loaded here.

Reply:
Yes, I see what you mean, now. I have added it back in.